app/javascript/src/components/Dashboard/Contacts/Table/index.jsx

```diff
@@ -0,0 +1,23 @@
+/* eslint-disable */
+import React from "react";
+import { Table as NeetoTable } from "neetoui";
+import { CONTACTS_LIST } from "../constants";
+import { contactsTableColumnData } from "./TableData";
+
+const Table = ({ setShowDeleteAlert }) => {
+  return (
+    <NeetoTable
+      columnData={contactsTableColumnData(setShowDeleteAlert)}
+      currentPageNumber={1}
+      defaultPageSize={10}
+      handlePageChange={function noRefCheck() {}}
+      onRowClick={function noRefCheck() {}}
+      onRowSelect={function noRefCheck() {}}
```

Comment:
https://academy.bigbinary.com/learn-react/pages/prefer-es6-arrow-functions#:~:text=Always%20prefer%20arrow%20functions

Ensure you go through the React Best Practices book very carefully. The point of creating such a book is to ensure we avoid unnecessary review comments and spend time on it, like I am doing right now.

Also make sure to go through the review comments I have put up in the Slack thread.

Reply:
Fixed it.